hubs/hubs.model.lkml

```diff
@@ -2,4 +2,4 @@ connection: "telemetry"
 label: "Hubs"
 include: "views/*"
 include: "explores/*"
-include: "dashboards/*"
```

Comment:
A fix has been made and you should not need to comment out the unrelated code anymore.  Can you try pulling from production and try again?

Reply:
Thanks! I undid these changes.